metadata-ingestion/src/datahub/ingestion/source/unity/ge_profiler.py

```diff
@@ -179,7 +229,9 @@ def get_unity_profile_request(
         return TableProfilerRequest(
             table=table,
             pretty_name=dataset_name,
-            batch_kwargs=dict(schema=table.ref.schema, table=table.name),
+            batch_kwargs=dict(
+                schema=table.ref.schema, table=table.name, catalog=table.ref.catalog
```

Comment:
mostly just so that it's clear what the hierarchy is

```suggestion
                catalog=table.ref.catalog, schema=table.ref.schema, table=table.name,
```